### PR TITLE
Add TransType 4 trial cask

### PR DESCRIPTION
**Pull request by gvost**
After making all changes to the cask:

- [ ] `brew cask audit --download {{cask_file}}` is error-free.
- [ ] `brew cask style --fix {{cask_file}}` reports no offenses.
- [ ] The commit message includes the cask’s name and version.

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [ ] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

**Comment by commitay** [comment]
> Free trial for 7 days

Can this be upgraded to the full version from within the app?

**Comment by gvost** [comment]
I am having a hard time finding a place to do that, I did a fresh install with the cask that I made. I remember that when I did buy it the first time I had the trial conclude there was an in-app prompt, which I believe linked out to the fontlab/transtype site.

**Comment by commitay** [comment]
>  linked out to the fontlab/transtype site.

Do you have to download the `app` again?

If there isn't a way to "unlock" the full version from within the trial, we'll refuse the Cask.

File: Casks/transtype-trial.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'transtype-trial' do
```
Comment (vitorgalvao) [inline Casks/transtype-trial.rb]: This does not follow the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).

**Comment by vitorgalvao** [comment]
@gvost You’re also supposed to do the top items from the checklist.

**Comment by gvost** [comment]
@commitay, the serial number is entered into the app, it is emailed to you. that unlocks the app. there definitely wasn't a need to re-download a "full" app.

**Comment by vitorgalvao** [comment]
> Can this be upgraded to the full version from within the app?

Yes.

image

**Comment by gvost** [comment]
@vitorgalvao, sorry about that, I must have misread that section title, I took it to mean those items needed to be checked off if I was submitting an update to the cask. On to your other comment, about the nomenclature for token reference, I think maybe I misunderstood there as well. So if it's not trouble, could you tell me if it would be correct for me to use `transtype-4-trial` to do the naming? Should also I then change the version from `:latest` to 4? I am looking at [this](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md#converting-to-ascii) section

**Comment by vitorgalvao** [comment]
@gvost What matters is that the token follows the name of the app bundle. In most cases, they’ll be equivalent.

File: Casks/transtype.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+cask 'transtype' do
```
Comment (gvost) [inline Casks/transtype.rb]: excellent! thanks that clarified it perfectly @vitorgalvao

**Comment by miccal** [comment]
Thank you for your contribution.

**Pull request merged**